src/Orleans.Runtime/Hosting/CoreHostingExtensions.cs

```diff
@@ -93,33 +93,33 @@ public static ISiloHostBuilder ConfigureLocalHostPrimarySilo(this ISiloHostBuild
         /// <summary>
         /// Configure silo to use GrainBasedMembership
```

Comment:
Rename the xml too?

Reply:
but that one still uses the legacy-based configuration... it wasn't updated to use the new config

Reply:
nevermind, I thought it was pointing to the ConfigureLocalHostPrimarySilo method... github confused me. I'll update